### DISCUSSION TITLE
Using `@rollup/plugin-wasm`

### DISCUSSION THREAD

**Discussion by brillout** [vikejs/vike#711] [Discussion] [answered]
Moved from Discord:

> is there a guide on how to integrate a rollup plugin with vite-plugin-ssr?

**Comment by nikivdev** [comment, marked as answer]
I want to add https://github.com/rollup/plugins/tree/master/packages/wasm to a solid app built with vite-plugin-ssr but hitting an issue.

edit: The issue is [not related to vite-plugin-ssr](https://github.com/rollup/plugins/discussions/1459)

**Reply by brillout** [reply]
Does it work with Vite without vite-plugin-ssr?

**Reply by nikivdev** [reply]
Apologise as I think this is not an issue with `vite-plugin-ssr`. So this discussion can be effectively closed. Thank you for reply.

**Reply by brillout** [reply]
👍 Thanks for the update.

Btw. in case your company is up for it: https://github.com/sponsors/brillout.

**Reply by nikivdev** [reply]
I believe Inlang is already sponsoring vite-plugin-ssr ♥️

**Reply by brillout** [reply]
Indeed 💚. I was actually wondering "somehow Nikita is ringing a bell" 😀. I'll add your GitHub username handle to the list of sponsors.

Thanks!
